shared/src/business/useCaseHelper/caseConfirmation/generateCaseConfirmationPdf.js

```diff
@@ -120,7 +120,7 @@ exports.generateCaseConfirmationPdf = async ({
       Body: result,
       Bucket: documentsBucket,
       ContentType: 'application/pdf',
-      Key: documentId,
+      Key: caseConfirmationPdfName,
```

Comment:
So... I left a note where this case confirmation pdf string was built. There don't seem to be checks to prevent it from returning, say, "case-NaN-confirmation.pdf" or something else strange if $this.docketNumber doesn't resolve correctly. That might cause problems here if you have multiple occasions where the pdf filename ends up used as a key into the document bucket.